examples/src/main/java/org/apache/spark/examples/ml/JavaANOVATestExample.java

```diff
@@ -51,7 +51,7 @@ public static void main(String[] args) {
     List<Row> data = Arrays.asList(
       RowFactory.create(3.0, Vectors.dense(1.7, 4.4, 7.6, 5.8, 9.6, 2.3)),
       RowFactory.create(2.0, Vectors.dense(8.8, 7.3, 5.7, 7.3, 2.2, 4.1)),
-      RowFactory.create(1.0, Vectors.dense(1.2, 9.5, 2.5, 3.1, 8.7, 2.5)),
+      RowFactory.create(3.0, Vectors.dense(1.2, 9.5, 2.5, 3.1, 8.7, 2.5)),
```

Comment:
I guess it's not a good example to have only one data in a category. I changed 1.0 -> 3.0, so there are two data in each of the category.